assets/src/scripts/components/hydrograph/time-series-data.js

```diff
@@ -0,0 +1,117 @@
+import { extent } from 'd3-array';
```

Comment:
This code (formly plotDataLine and plotDataLines) was pulled out of time-series-graph so that it can also be used in graph-brush. I did not add tests as the code is somewhat tested by tests in time-series-graph.